extensions/functions_rounding.yaml

```diff
@@ -27,3 +27,92 @@ scalar_functions:
           - value: fp64
             name: "x"
         return: fp64
+  -
+    name: "round"
+    description: >
+      Rounding the value `x` to `s` decimal places.
+    impls:
+      - args:
+          - value: fp32
```

Comment:
Are there other pairs of argument types that we want to support?  I could see fp64 and i8 as potential options here.  With the ability to round anywhere in the number we may also want to include integer types for x.

Reply:
Not entirely sure about `i8`, although with s being `i32` we have already added `fp32` and `fp64`. 
cc @westonpace

Reply:
For the moment let's assume it is safe to mandate that `s` be `i32`.  If we need to add additional kernels for `s` later then we can do so without much strife I hope.

However, we should allow `x` to be `i8, i16, i32, i64`.  So please add the additional 4 kernels.